external/a2a/turbo.json

```diff
@@ -0,0 +1,15 @@
+{
+  "extends": ["//"],
+  "tasks": {
+    "build": {
+      "inputs": ["$TURBO_DEFAULT$", ".env*"],
+      "outputs": [".next/**", "!.next/cache/**"],
+      "cache": false,
+      "dependsOn": [
+        "@microsoft/teams.common#build",
+        "@microsoft/teams.apps#build",
+        "@microsoft/teams.ai#build"
```

Comment:
I don't think this is needed for ai since you aren't using it as a dependency in your package.json.

Reply:
Added dep in package.json